4.7.0 build folder has no build file

Hi there,
- What you're trying to do
I was doing an "npm install" of this package and the build folder
`@google-cloud/bigquery-storage/build`
appeared to have only the `package.json` file. All import statements (Typescript) that referred to this module reported that 
> Cannot find module '@google-cloud/bigquery-storage' or its corresponding type declarations.ts(2307)

- What code you've already tried
I checked the npm repository's files and it did look strange:

![Screenshot 2024-05-09 at 09 35 45](https://github.com/googleapis/nodejs-bigquery-storage/assets/149639496/8e2255c5-ca4b-488f-a7d9-cf840776cdd7)

I switched to use the fixed version 4.6.0 and more files became available in the `build` foler. 

- Any error messages you're getting
All builds of my service that were built successfully some days ago are now failing because of that. 

Can someone have a look please? The issue seems just that build files weren't packaged/uploaded onto 4.7.0 on npm. 

Regards,
Lee

--- a/system-test/managed_writer_client_test.ts
+++ b/system-test/managed_writer_client_test.ts
@@ -13,6 +13,8 @@
 // limitations under the License.
 
 import * as assert from 'assert';
+import {readFileSync} from 'fs';
+import * as path from 'path';
 import {describe, it} from 'mocha';
 import * as uuid from 'uuid';
 import * as gax from 'google-gax';
@@ -25,7 +27,10 @@ import {ClientOptions} from 'google-gax';
 import * as customerRecordProtoJson from '../samples/customer_record.json';
 import {JSONEncoder} from '../src/managedwriter/encoder';
 import {PendingWrite} from '../src/managedwriter/pending_write';
-import * as pkg from '../package.json';
+
+const pkg = JSON.parse(
+  readFileSync(path.resolve(__dirname, '../../package.json'), 'utf-8')
+);
 
 const sandbox = sinon.createSandbox();
 afterEach(() => sandbox.restore());

